### PR TITLE
Add missing migration

### PR DESCRIPTION
**Pull request by magajh**
This PR adds the missing migration from the changes made when adding support for juniper.

### CODE REVIEW

**Review by andrey-canon — APPROVED** [review]

File: eox_audit_model/tests/tests_models.py [hunk 1]
```diff
@@ -32,15 +33,18 @@ def test_valid_request(self, get_current_request_mock):
             - get_current_request is called once.
         """
         request = Mock()
-        expected_value = 'https://howgarts.com'
+        domain = 'https://howgarts.com'
+        site = Site.objects.create(
+            domain=domain
```
Comment (andrey-canon) [inline eox_audit_model/tests/tests_models.py:38]: trailing comma

File: eox_audit_model/tests/tests_models.py [hunk 2]
```diff
@@ -52,11 +56,13 @@ def test_invalid_request(self, get_current_request_mock):
             - get_current_request is called once.
         """
         get_current_request_mock.return_value = None
-        expected_value = 'Missing domain.'
+        site = Site.objects.create(
+            domain='Missing domain.'
```
Comment (andrey-canon) [inline eox_audit_model/tests/tests_models.py:60]: trailing comma

**Review by andrey-canon — CHANGES_REQUESTED** [review]

File: eox_audit_model/tests/tests_models.py (hunk 1 above)
Comment (magajh) [reply eox_audit_model/tests/tests_models.py]: Done

File: eox_audit_model/tests/tests_models.py (hunk 2 above)
Comment (magajh) [reply eox_audit_model/tests/tests_models.py]: Done

**Review by andrey-canon — APPROVED** [review]

**Pull request merged**